### PR TITLE
PR for Shin branch

### PR DESCRIPTION
**Pull request by m-walters**
Organized some things, other housekeeping

Of note:
- moved some `main()` logic out for clarity.
- logger doesn't need to be passed through functions

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,13 +1,89 @@
...
-import wandb
-from ufm import countermeasures, fine_tuning, metrics, models, pretrain_score, unadapt, utils
+from ufm import countermeasures, fine_tuning, metrics, models, unadapt, utils
+
+
+def run_baseline_suite():
+    """
+    Run the suite of methods and metrics for a baseline model
+    """
+    logger = logging.getLogger()
```
Comment (mukobi) [inline main.py:16]: Would recommend `logging.getLogger(__name__)`
https://docs.python.org/3/library/logging.html#logger-objects

File: main.py [hunk 2]
```diff
@@ -1,13 +1,89 @@
...
+    metrics.run_benchmark(model_base, "finetuned")
+
+    logger.info("Calculating final unadaptability metrics")
+    loss_gap_ratio = metrics.calculate_unadaptability_metrics(
+        ft_val_losses, wandb.config.baseline_metrics_path, wandb.config.model
+    )
+
+    logger.info("This is a baseline run, loss gap ratio should be zero.")
+    logger.info(f"Final loss gap ratio: {loss_gap_ratio:.6f}")
+    logger.info("Finished Baseline suite")
```
Comment (mukobi) [inline main.py:42]: Maybe you could log an error if it's not close to 0? Not too important though if you'd rather not

File: src/ufm/countermeasures.py [hunk 3]
```diff
@@ -3,6 +3,8 @@
 import torch
 import torch.nn as nn
 
+logger = logging.getLogger()
```
Comment (mukobi) [inline src/ufm/countermeasures.py:6]: `logging.getLogger(__name__)` again

Comment (mukobi) [reply src/ufm/countermeasures.py:6]: Also, this probably should _not_ be in the top level of this file and instead stored as a variable in each class on init

**Review by ShinnosukeUesaka — APPROVED** [review]
LGTM

**Pull request merged**